.github/workflows/go.yml

```diff
@@ -26,3 +26,28 @@ jobs:
 
     - name: Build
       run: go build -v .
+      
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
```

Comment:
why an older version? any necessity here?

Reply:
OK, I'll fix it.